### PR TITLE
Fix host content virtwho

### PR DESCRIPTION
**Pull request by sambible**
Revert/rework of https://github.com/SatelliteQE/robottelo/pull/9831 - this PR removed "hypervisorId" from the virtwho helper, and resulted in the tests mentioned there still passing. This unfortunately broke another test, in test_contenthosts.py

This readds that portion of the JSON payload, based on comments in this BZ - https://bugzilla.redhat.com/show_bug.cgi?id=2083444

All tests will still pass, I'll get PRT in here to confirm. 

(Note: This got mixed in with commits from another PR, so I'll need to resolve that/get that PR Merged first. It is already approved, so it should be fine)

### CODE REVIEW

**Review by pondrejk — APPROVED** [review]

File: robottelo/utils/virtwho.py [hunk 1]
```diff
@@ -468,7 +468,9 @@ def hypervisor_json_create(hypervisors, guests):
                     "attributes": {"active": 1, "virtWhoType": "esx"},
                 }
             )
-        hypervisor = {"guestIds": guest_list}
+        hypervisor = {"hypervisorId": str(uuid.uuid4()), "guestIds": guest_list}
```
Comment (jyejare) [inline robottelo/utils/virtwho.py:471]: I dont see `hypervisor` is not being reused below and we overriding hypervisor variable from line 473.

Can you please check that and fix it right ?

**Review by jyejare — CHANGES_REQUESTED** [review]

File: robottelo/utils/virtwho.py (hunk 1 above)
Comment (sambible) [reply robottelo/utils/virtwho.py:471]: Oh yeah, i left this line in. Lemme fix that. Good catch :)

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contenthost.py -k "test_search_for_virt_who_hypervisors"

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/virtwho/cli/test_esx.py -k test_positive_post_hypervisors

**Review by ogajduse — APPROVED** [review]

**Review by jyejare — APPROVED** [review]

**Pull request merged**